recipes/r-canopy/meta.yaml

```diff
@@ -46,15 +43,7 @@ requirements:
 
 test:
   commands:
-    # You can put additional test commands to be run here.
-    - $R -e "library('Canopy')"  # [not win]
-    - "\"%R%\" -e \"library('Canopy')\""  # [win]
-
-  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
-  # in the recipe that will be run at test time.
-
-  # requires:
-    # Put any additional test requirements here.
+    - $R -e "library('Canopy')"
```

Comment:
This wouldn't be needed (see above).

Reply:
O.k. I got the impression that it's not going to run on win. I re-added the directives and removed the comments.